javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,26 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
+
+function renderProducts(products) {
```

Comment:
If you want to make dynamic list, you need to clear `productsUl`  otherwise it will be static ex:`productsUl.innerHTML = ''`